src/release/release-notes/release-notes-3.16.0.md

```diff
@@ -0,0 +1,938 @@
+---
+title: Flutter 3.16.0 release notes
+short-title: 3.16.0 release notes
+description: Release notes for Flutter 3.16.0.
+---
+This page has release notes for 3.16.0.
+For information about subsequent bug-fix releases,
+see [Hotfixes to the Stable Channel][].
+
+[Hotfixes to the Stable Channel]: {{site.github}}/flutter/flutter/wiki/Hotfixes-to-the-Stable-Channel
+
+## Framework
+
+### Framework
```

Comment:
The double "Framework" headers are a bit confusing. Any idea on what's intended here?

Reply:
> The double "Framework" headers are a bit confusing. Any idea on what's intended here? Perhaps the first one should be "SDK" and "Framework" is a subset of that, just like "Tools"?

Yeah, looking for adjustments here.  Maybe title the first one Flutter instead of framework. 

Next cycle https://github.com/flutter/flutter/pull/136967 will be picked up by the first candidate allowing for the labels we created a few months ago.